presign/base/models.py

```diff
@@ -329,6 +321,15 @@ def get_answers(self):
             "question__order",
         )
 
+    def get_messages(self, current=True):
+        if current:
+            filtered = ParticipantCustomMessage.objects.filter(
+                participant=self, is_current=True
+            )
+        else:
+            filtered = ParticipantCustomMessage.objects.filter(participant=self)
+        return filtered.order_by("-created_at")
+
```

Comment:
the `order_by` should probably be in the respective meta class attribute of the custom message class.

Reply:
Done